proposals/aip-1.md

```diff
@@ -0,0 +1,229 @@
+
+
+# Open Issues / TODO #
+- [x] ~~Near the mention of "We use GitHub's [required reviews for pull requests]" above, but that is for every review/pull of the PR. PRs can be merged in a Draft status without being accepted. More final status (Accepted, Deferred, Rejected, and Replaced, Final) require more careful reviews. That needs clarified. Or we could make a different directory for Draft proposals and Accepted/Finalized/Active/Replaced proposals to keep the review/approval requirement lower for Draft proposals.~~
+    - It can be difficult to easily preserve and view file history across renames, so I think we just require n of M board approvals for ALL pulls and make sure that they are trained to quite liberally merge draft proposals and very judiciously merge proposals with any other "final" status. This also keeps the URL to an AIP intact regardless of status.
+- [ ] Set up the [@Agorex-io/proposal-editors] team.
+- [ ] Set up the [@Agorex-io/board-of-directors] team.
+  - Initially proposal-editors and board-of-directors are likely to be the same members. Long term this may scale better to keep them separate.
+- [ ] README.md
+- [ ] add PULL_REQUEST_TEMPLATE.md
+- [ ] Document and link to "Agorex values" above.
+- [x] Double check by searching for PEP, EIP and Ethereum in doc content.
+- [x] Double check by searching for Meta -> Proposal in doc content.
+- Low priority:
+  - [ ] Setup github pages on the repo. Necessary?
+
+
+# Future Consideration #
+- Use PGP-signed commits?
+
+
+# References #
+  [@Agorex-io/board-of-directors]: https://github.com/orgs/Agorex-io/teams/board-of-directors
+  [Agorex Board of Directors]: https://github.com/orgs/Agorex-io/teams/board-of-directors
+  [@Agorex-io/proposal-editors]: https://github.com/orgs/Agorex-io/teams/proposal-editors
+  [the Issues section of this repository]: https://github.com/Agorex-io/Proposals/issues
+  [the Agorex subreddit]: https://www.reddit.com/r/Agorex/
+  [the Agorex Discord chat rooms]: https://discord.gg/bEPhszu
+  [pull request in this repository]: https://github.com/Agorex-io/Proposals/pulls
+  [required reviews for pull requests]: https://help.github.com/articles/about-required-reviews-for-pull-requests/
+  [Enabling required reviews for pull requests (GitHub)]: https://help.github.com/articles/enabling-required-reviews-for-pull-requests/
+  [RFC 822]: http://www.faqs.org/rfcs/rfc822.html
+  [The Ethereum Community's EIP-1]: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md
+  [The Python Community's PEP 1]: https://www.python.org/dev/peps/pep-0001/
+  [Ethereum's EIP 1]: https://github.com/ethereum/EIPs/blob/a90a5a3e58accea956dd8f1eacac5a17f98ec2e2/EIPS/eip-1.md
+  [markdown]: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet
+  [Bitcoin's BIP-0001]: https://github.com/bitcoin/bips
+  [Python's PEP-0001]: https://www.python.org/dev/peps/
+
```

Comment:
We need something down here talking about how this proposal process is largely derived from...